tooling/lib/get-mediatype-examples.js

```diff
@@ -0,0 +1,83 @@
+const { sampleFromSchema } = require('../samples');
+const cleanStringify = require('./json-stringify-clean');
+
+module.exports = {
+  /**
+   * Extracts an example from an OAS Media Type Object. The example will either come from the `example` property, the
+   * first item in an `examples` array, or if none of those are present it will generate an example based off its schema.
+   *
+   * @link https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#mediaTypeObject
+   * @param {object} mediaType
+   * @returns {(object|false)}
+   */
+  getMediaTypeExample: mediaType => {
+    if (mediaType.example) {
+      return mediaType.example;
+    } else if (mediaType.examples) {
+      const examples = Object.keys(mediaType.examples);
+      if (examples.length) {
+        if (examples.length > 1) {
+          // Since we're trying to return a single example with this method, but have multiple present,
+          // return `false` so `getMultipleExamples` will pick up this response instead later.
```

Comment:
`getMultipleExamples` is [here](https://github.com/readmeio/api-explorer/blob/ad91b045c493c92b1ae61102e4a25aa150e51fda/packages/api-explorer/src/lib/get-response-examples.js#L118) and [here](https://github.com/readmeio/api-explorer/blob/ad91b045c493c92b1ae61102e4a25aa150e51fda/packages/api-explorer/src/lib/get-response-examples.js#L55) right?

Reply:
@darrenyong yep